exercises/tests/tests4.rs

```diff
@@ -0,0 +1,40 @@
+// tests4.rs
+// Correct the tests to
+// Do not change Rectangle::new body
+// Execute `rustlings hint tests4` or use the `hint` watch subcommand for a hint.
+
+// I AM NOT DONE
+
+struct Rectangle {
+    width: i32,
+    height: i32
+}
+
+impl Rectangle {
+    pub fn new(width: i32, height: i32) -> Self {
+        if width < 0 || height < 0 {
+            panic!("Rectangle width and height cannot be negative!")
+        }
+        Rectangle {width, height}
+    }
+}
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn correct_width_and_height() {
```

Comment:
Is there any better way of expression what we want those tests to do? Maybe add a comment at the beginning of each test saying something like `This test should check if the rectangle is the size that we pass into its constructor`, and same for the other tests.